### PR TITLE
bug 1442745: upgrade bleach to 2.1.3

### PR DESCRIPTION
**Pull request by g-k**
Upgrades bleach to 2.1.3 the hashes are for the 2.1.3 tgz and whl files respectively.

### CODE REVIEW

**Comment by jwhitlock** [comment]
I'm going to fix a spelling error in the tests, and split into an invalid and valid href test

File: kuma/wiki/tests/test_content.py [hunk 1]
```diff
@@ -694,25 +695,48 @@ def test_filteriframe_default_rejected(url):
...
+
+BLEACH_VALID_HREFS = {
+    'relative': '/docs/ok/test',
+    'http': 'http://example.com/docs/ok/test',
+    'https': 'https://example.com/docs/ok/test',
+}
+
+
+@pytest.mark.parametrize('href', BLEACH_INVALID_HREFS.values(),
+                         ids=BLEACH_INVALID_HREFS.keys())
```
Comment (willkg) [inline kuma/wiki/tests/test_content.py]: OMG! ids is amazing! I had no idea that existed!

File: kuma/wiki/tests/test_content.py [hunk 2]
```diff
@@ -694,25 +695,48 @@ def test_filteriframe_default_rejected(url):
...
+                         ids=BLEACH_VALID_HREFS.keys())
+def test_bleach_clean_hrefs(href):
+    """Bleach retains valid hrefs."""
+    html = '<p><a id="test" href="%s">click me</a></p>' % href
+    result = bleach.clean(html,
+                          tags=ALLOWED_TAGS,
+                          attributes=ALLOWED_ATTRIBUTES,
+                          protocols=ALLOWED_PROTOCOLS)
+    link = pq(result).find('#test')
+    assert link.attr('href') == href
```
Comment (willkg) [inline kuma/wiki/tests/test_content.py]: This is a nice cleanup of the previous test!

**Comment by willkg** [comment]
r_plus

**Pull request merged**